CHANGELOG.md

```diff
@@ -37,6 +42,10 @@ All notable changes to this project will be documented in this file. The format
 
 [Full Changelog](https://github.com/puppetlabs/puppetlabs-facter_task/compare/0.3.1...0.4.0)
 
+### Changed
```

Comment:
@david22swan looks like this PR is added to v0.5.0 instead of v0.6.0.Could you please check